src/integrations/admin/indexing-tool-integration.php

```diff
@@ -166,7 +167,7 @@ public function enqueue_scripts() {
 		$data = [
 			'disabled'  => ! $this->indexable_helper->should_index_indexables(),
 			'amount'    => $this->get_unindexed_count(),
-			'firstTime' => ( $this->options_helper->get( 'indexing_first_time', true ) === true ),
+			'firstTime' => ( $this->indexing_helper->get_first_time() === true ),
```

Comment:
If you take over my suggestion above, this can be replaced by
```suggestion
			'firstTime' => ( $this->indexing_helper->is_first_time() ),
```

Reply:
Done